src/ngraph/op/fused/lstm_sequence.hpp

```diff
@@ -0,0 +1,154 @@
+            }
+
+            explicit LSTMSequence(const Output<Node>& X,
+                                  const Output<Node>& initial_hidden_state,
+                                  const Output<Node>& initial_cell_state,
+                                  const Output<Node>& sequence_lengths,
+                                  const Output<Node>& W,
+                                  const Output<Node>& R,
+                                  const Output<Node>& B,
+                                  const std::int64_t hidden_size,
+                                  const std::string direction,
+                                  const std::vector<float> activations_alpha = {},
+                                  const std::vector<float> activations_beta = {},
+                                  const std::vector<std::string> activations = {},
+                                  const float clip_threshold = 0,
+                                  const bool input_forget = false)
+                : LSTMSequence(X,
+                               initial_hidden_state,
+                               initial_cell_state,
+                               sequence_lengths,
+                               W,
+                               R,
+                               B,
+                               Constant::create(element::f32, Shape{}, std::vector<float>{0.f}),
+                               hidden_size,
+                               direction,
+                               activations_alpha,
+                               activations_beta,
+                               activations,
+                               clip_threshold,
+                               input_forget)
+            {
+            }
+
+            void validate_and_infer_types() override;
+
+            virtual NodeVector decompose_op() const override;
+
+            virtual std::shared_ptr<Node>
+                copy_with_new_args(const NodeVector& new_args) const override;
+
+            std::vector<float> get_activations_alpha() const { return m_activations_alpha; }
+            std::vector<float> get_activations_beta() const { return m_activations_beta; }
+            std::vector<std::string> get_activations() const { return m_activations; }
+            float get_clip_threshold() const { return m_clip_threshold; }
+            std::string get_direction() const { return m_direction; }
+            std::int64_t get_hidden_size() const { return m_hidden_size; }
+            bool get_input_forget() const { return m_input_forget; }
+        private:
+            ///
+            /// \brief      Gets the masked node according to sequence lenght in a batch.
+            ///
+            /// \note       Zeros out values or sets them to default value for inputs with
+            ///             sequence lenght shorter than currently procssed time step.
+            ///
+            /// \param[in]  data           The input node.
```

Comment:
Use `value` instead of `node`. Nodes can have more than one output, so these are not nodes.
If you are calling it the input in the doc, use `input` for the variable name.

Reply:
Done